assets/js/atomic/blocks/product-elements/price/attributes.js

```diff
@@ -12,14 +11,9 @@ let blockAttributes = {
 		type: 'boolean',
 		default: false,
 	},
+	textAlign: {
+		type: 'string',
+	},
 };
 
-if ( isFeaturePluginBuild() ) {
```

Comment:
Out of curiosity: why are we removing the feature flag here? Was this part of a decision documented elsewhere?

Reply:
No idea why it was there in the first place. I have asked @Aljullu, but he didn't know either. It doesn't really make sense for it to be there as `Text Alignment` is not an experimental feature. We have decided it's safe to just remove it.

Also that setting was broken on the frontend anyway 🤷‍♂️

Reply:
That makes sense: thanks for clarifying!